groups/alteryx-sandbox/security_groups.tf

```diff
@@ -92,6 +92,14 @@ resource "aws_vpc_security_group_ingress_rule" "ec2_worker_api" {
   to_port                      = 80
 }
 
+
+resource "aws_vpc_security_group_egress_rule" "ec2_egress" {
+  security_group_id = aws_security_group.ec2.id
+
+  cidr_ipv4   = "0.0.0.0/0"
+  ip_protocol = "All"
```

Comment:
I don't believe this is valid. `ip_protocol` should be `-1`

Reply:
Have updated.